omop_file_validator.py

```diff
@@ -438,13 +522,171 @@ def run_checks(file_path, f):
+
+                if meta_column_name not in row.columns and meta_column_required:
+                    result['errors'].append(
+                        dict(message='Missing required column',
+                             column_name=meta_column_name))
+                    row_error_found = True
+                elif pd.isnull(
+                        row[meta_column_name].loc[0]) and meta_column_required:
+                    result['errors'].append(
+                        dict(message=MSG_NULL_DISALLOWED,
+                             column_name=meta_column_name))
+                    row_error_found = True
+                else:
+                    value = row[meta_column_name].loc[0]
+                    row_column_type = row[meta_column_name].dtype
+
+                    if find_error_in_row(row, meta_column_name,
+                                         meta_column_type):
+                        e = dict(message=MSG_INVALID_TYPE + " line number " +
+                                 str(idx + 1),
+                                 column_name=meta_column_name,
+                                 actual=value,
+                                 expected=meta_column_type)
+                        result['errors'].append(e)
+                        row_error_found = True
+
+                    # Check that date format is in the YYYY-MM-DD or YYYY-MM-DD hh:mm:ss format
+                    if meta_column_type in ('date', 'timestamp'):
+                        fmts = ''
+                        err_msg = ''
+
+                        if meta_column_type == 'date':
+                            fmts = VALID_DATE_FORMAT
+                            err_msg = MSG_INVALID_DATE
+                        elif meta_column_type == 'timestamp':
+                            fmts = VALID_TIMESTAMP_FORMAT
+                            err_msg = MSG_INVALID_TIMESTAMP
+
+                        if not any(
+                                list(
+                                    map(
+                                        lambda fmt: date_format_valid(
+                                            str(value), fmt), fmts))):
+                            e = dict(message=err_msg + ": line number " +
+                                     str(idx + 1),
+                                     column_name=meta_column_name,
+                                     actual=value,
+                                     expected=meta_column_type)
+                            result['errors'].append(e)
+                            row_error_found = True
+
+    except Exception as e:
+        print(traceback.format_exc())
+        # Adding error message if there is a wrong number of columns in a row
+        # result['errors'].append(dict(message=e.args[0].rstrip()))
+        # row_error_found = True
+    else:
+        print(
+            'CSV file for "%s" parsed successfully. Please check for errors in the results files.'
```

Comment:
This should be JSONL
```suggestion
            f'JSONL file for "{table_name}" parsed successfully. Please check for errors in the results files.'
```